### PR TITLE
Feature/427 geo area search

### PR DESCRIPTION
**Pull request by zarnifoulette**
- Search is triggered when a suggestion is selected. 
- URIs are passed to the kernel.

### CODE REVIEW

File: src/main/resources/public/js/components/autosuggests/geoarea-autosuggest.js [hunk 1]
```diff
@@ -58,20 +58,29 @@ class GeoAreaAutosuggest extends Component {
...
+    getSuggestionValue = suggestion => {
+        return suggestion.name || GeoAreaAutosuggest.defaultProps.value;
+    };
 
     shouldRenderSuggestions = (input) => {
         return input && (input.trim().length >= sizeQueryBeforeFetch);
     };
 
+    handleOnChange = (event, {newValue}) => {
+        GeoAreaAutosuggest.defaultProps.value = newValue;
```
Comment (bobeal) [inline src/main/resources/public/js/components/autosuggests/geoarea-autosuggest.js]: A default prop should not change, it is only meant to hold a default value when none has been specified in props.value

**Comment by bobeal** [comment]
@zarnifoulette did you check that the other uses of GeoAreaAutosuggest had no regressions ?

**Comment by zarnifoulette** [comment]
Yes I checked that (profile, organization, store), and no regressions were detected.

**Comment by zarnifoulette** [comment]
I just have one problem when I want to save new properties of an organization an error is raised.This error is raised when the server try to connect to the DataCore (revceive 400).

**Pull request merged**